geolocator_android/lib/src/types/android_settings.dart

```diff
@@ -34,12 +37,17 @@ class AndroidSettings extends LocationSettings {
   /// If this value is `null` an interval duration of 5000ms is applied.
   final Duration? intervalDuration;
 
+  /// If this is set then the services is started as a Foreground service with a persistent notification
+  /// showing the user that the service will continue running in the background
```

Comment:
Nit: sentence should end with a period.
```suggestion
  /// showing the user that the service will continue running in the background.
```